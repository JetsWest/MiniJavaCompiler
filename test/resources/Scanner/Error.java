x = 5;
if (x > 4) {
    x = 6;
}