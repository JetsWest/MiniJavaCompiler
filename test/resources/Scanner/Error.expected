ID(x) BECOMES DIGIT(5) SEMICOLON IF LPAREN ID(x) <UNEXPECTED(>)> DIGIT(4) RPAREN LBRACE ID(x) BECOMES DIGIT(6) SEMICOLON RBRACE